frontend/package.json

```diff
@@ -110,7 +110,7 @@
         "stream-browserify": "^3.0.0",
         "style-loader": "^3.2.1",
         "swr": "^1.0.1",
-        "temptifly": "^0.4.28",
+        "temptifly": "^0.5.13",
```

Comment:
Not sure we should be making this big of a jump in the 2.4.z stream. @jeswanke has that fix been backported to temptifly 0.4.z?

Reply:
I'll give that a try if @jeswanke hasn't already? Will add you both on the PR.

Reply:
@Randy424 the 0.5.* is the temptifly for 2.5--you need to wait till I add the immutable stuff to 0.4.* before you can use in 2.4---there are differences between the two